recipes/snownlp/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "snownlp" %}
+{% set version = "0.12.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/snownlp-{{ version }}.tar.gz
+  sha256: c92accd025b70dd16706a10690f556ac9204bb6189f7dc68ece5c207c9bc27d8
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - snownlp
+    - snownlp.classification
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/isnowfy/snownlp
+  summary: Python library for processing Chinese text
+  license: MIT
+  license_file: LICENSE.md
```

Comment:
Hi, can you please add [these fields](https://github.com/izahn/staged-recipes/blob/snownlp/recipes/example/meta.yaml#L76-L82) too? Thanks!

Reply:
I added a description, but as far as I can tell there is no separate documentation or dev url; should I list https://github.com/isnowfy/snownlp in those fields as well or leave them out?

Reply:
I normally just add them anyway even if they are the same, but no need. LGTM.